numcodecs/tests/test_zarr3.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+from typing import Iterator
+
+
+import numpy as np
+import pytest
+import sys
+
+from numcodecs.registry import get_codec
+
+try:
+    from zarr.codecs.registry import get_codec_class
+    from zarr.array import Array
+    from zarr.common import JSON
+    from zarr.codecs import BytesCodec
+    from zarr.abc.store import Store
+    from zarr.store import MemoryStore, StorePath
+
+except ImportError:
+    pass
```

Comment:
Should this whole try; except block be replaced by a pytest skip that skips if zarr<3 is installed?